content/en/docs/concepts/containers/images.md

```diff
@@ -75,7 +119,17 @@ For example, if you create a Deployment with an image whose tag is _not_
 the pull policy of any object after its initial creation.
 {{< /note >}}
 
-When `imagePullPolicy` is defined without a specific value, it is also set to `Always`.
+#### Required image pull
+
+If you would like to always force a pull, you can do one of the following:
+
+- set the `imagePullPolicy` of the container to `Always`.
```

Comment:
Is there a reason the first letter here is not capitalized in all the list items? Recommend capitalizing.

Reply:
We usually write this kind of list all lower case, but it makes sense here because each item is a sentence. Good spot.